frontend/src/editor/Cell.tsx

```diff
@@ -186,18 +181,25 @@ const CellComponent = (
     [editorView, prepareToRunEffects]
   );
 
-  // Hack to provide the value of `running` to Code Mirror's EditorView.
```

Comment:
this was as small drive-by change. but using `useEvent`, we pass the latest value of loading